lib/utils.js

```diff
@@ -647,8 +647,6 @@ exports.stackTraceFilter = function() {
   function isMochaInternal(line) {
     return (
       ~line.indexOf('node_modules' + slash + 'mocha' + slash) ||
-      ~line.indexOf('node_modules' + slash + 'mocha.js') ||
-      ~line.indexOf('bower_components' + slash + 'mocha.js') ||
       ~line.indexOf(slash + 'mocha.js')
```

Comment:
both L650 and L651 are unnecessary because of L652

Reply:
@boneskull, [your #2503 updates](https://github.com/mochajs/mocha/commit/8bf1f297142b02128664523ab5a9b3d55b09c365#diff-50e3aa130a4f97a42ee2cf111c7b1d9d) don't appear to be equivalent changes for these lines...

The two lines appear to need correction, not removal.

Reply:
my changes to these two lines in 8bf1f297142b02128664523ab5a9b3d55b09c365 were incorrect, but they are also extraneous; the lines I replaced were *also* extraneous.  removing them is correct.